spec/features/training_tool_spec.rb

```diff
@@ -101,6 +101,17 @@
       expect(tmu.reload.last_slide_completed).to eq(module_2.slides.second.slug)
     end
 
+    it 'allows for navigation with the left and right buttons' do
+      click_link 'Start'
+      sleep 1.5
+      tmu = TrainingModulesUsers.find_by(user_id: user.id, training_module_id: module_2.id)
```

Comment:
What's this for?

Reply:
Oh, I see... copied from the other examples.

I think the sleep is only necessary because we want to find a record that got created (and later, updated) when the user clicked start. But for just testing navigation, we can skip anything to do with the database altogether, and also avoid the sleep delay.